package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "diaspora",
+  "version": "0.7.17.0",
```

Comment:
Oh, I just noticed the version here, is this needed for what we use this for? Is this something we also need to update for each release? Or can we leave this blank?

(as this will be merged to `develop` only, as it adds a new requirement for having yarn installed, it should probably be 0.7.99.0 if it's really needed to have a version here)

Reply:
`name` and `version` are the only required fields, and [it needs to be parsable by `node-semver` ](https://docs.npmjs.com/cli/v8/configuring-npm/package-json#version). But since this will never be published to npm, there is on harm in setting this to `0.0.0`.

Reply:
"If you don't plan to publish your package, the name and version fields are optional." :thinking: so maybe it's not required in our case? But also setting it to `0.0.0` sounds good to me.